docs/sources/datasources/influxdb/_index.md

```diff
@@ -0,0 +1,156 @@
++++
+title = "InfluxDB data source"
+description = "Guide for using InfluxDB in Grafana"
+keywords = ["grafana", "influxdb", "guide", "flux"]
+aliases = ["/docs/grafana/latest/features/datasources/influxdb", "/docs/grafana/latest/datasources/influxdb"]
+weight = 700
++++
+
+# InfluxDB data source
+
+{{< docs/shared "influxdb/intro.md" >}}
+
+This topic explains options, variables, querying, and other options specific to this data source. Refer to [Add a data source]({{< relref "add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.
+
+## Data source options
+
+To access data source settings, hover your mouse over the **Configuration** (gear) icon, then click **Data Sources**, and then click the data source.
```

Comment:
```suggestion
To access data source settings, hover your mouse over the **Configuration** (gear) icon, then click **Data sources**, and then click the data source.
```
Using sentence case going forward and the UI can catch in the context of plugin documentation.